sys/include/base64.h

```diff
@@ -8,9 +8,7 @@
  */
 
 /**
- * @defgroup    sys_base64 base64 encoder decoder
```

Comment:
Why these changes, IIRC I had this discussion before and we concluded to use `doc.txt` for group definition only when there is no proper header file. However, here we have these header files, so no need to introduce `doc.txt` files, too.

Reply:
I mean for `sys_base64`, `sys_ubjson` and the like

Reply:
hmmmm, I could revert this change. I was adding doc.txt files because it's easier to write per-module documentation in a doc file rather than headers. Shall I revert?

Reply:
ping @smlng

Reply:
as said I had this discussion in one of my own PRs and the consensus was to use header files where possible and limit the usage of `doc.txt` to cases where no header file is available for the module or group.

TL;DR: in this case I would suggest to revert, yes.

Reply:
@miri64 IIRC we had this discussion, any thought/comments here?

Reply:
I agree that adding `doc.txt` when not actually needed is not the way to go. The documentation stays with the code this way and isn't "hidden" in some file nobody ever really looks at. ;-)

Reply:
ok, I will revert them :)